类型检查失败: cooldown-manager.test.ts 缺少 vitest 类型引用，command-registry.test.ts 缺少 mock 方法

## 测试输出错误信息

运行 \`npm run type-check\` 失败：

### 问题 1: cooldown-manager.test.ts 缺少 vitest 类型引用
\`\`\`
src/schedule/cooldown-manager.test.ts(12,1): error TS2582: Cannot find name 'describe'. Do you need to install type definitions for a test runner?
src/schedule/cooldown-manager.test.ts(17,3): error TS2304: Cannot find name 'beforeEach'.
src/schedule/cooldown-manager.test.ts(23,3): error TS2304: Cannot find name 'afterEach'.
src/schedule/cooldown-manager.test.ts(28,5): error TS2582: Cannot find name 'it'. Do you need to install type definitions for a test runner?
src/schedule/cooldown-manager.test.ts(30,7): error TS2304: Cannot find name 'expect'.
... (共 100+ 个类型错误)
\`\`\`

### 问题 2: command-registry.test.ts mock 缺少新方法
\`\`\`
src/nodes/commands/command-registry.test.ts(17,3): error TS2739: Type '...' is missing the following properties from type 'CommandServices': getScheduleCooldownStatus, clearScheduleCooldown
\`\`\`

## 根本原因分析

这两个问题都是由 PR #874 (feat(schedule): add cooldown period for scheduled tasks) 引入的：

1. **cooldown-manager.test.ts**: 新创建的测试文件没有添加 vitest 全局类型引用三斜线指令 \`/// <reference types=\"vitest/globals\" />\`

2. **command-registry.test.ts**: PR #874 在 \`CommandServices\` 类型中新增了 \`getScheduleCooldownStatus\` 和 \`clearScheduleCooldown\` 方法，但 \`command-registry.test.ts\` 中的 mock 对象没有同步更新

## 相关代码变更

来自 commit d91a209:
\`\`\`
src/nodes/command-services.ts                   |   2 +
src/nodes/commands/commands/schedule-command.ts |  89 ++++++-
src/nodes/commands/schedule-command.test.ts     |  11 +-
src/nodes/commands/types.ts                     |  14 ++
src/nodes/schedule-management.ts                |  26 +++
src/nodes/scheduler-service.ts                  |   8 +
src/schedule/cooldown-manager.test.ts           | 189 +++++++++++++++
src/schedule/cooldown-manager.ts                | 296 ++++++++++++++++++++++++
\`\`\`

## 修复建议

1. 在 \`src/schedule/cooldown-manager.test.ts\` 文件开头添加:
   \`\`\`typescript
   /// <reference types=\"vitest/globals\" />
   \`\`\`

2. 在 \`src/nodes/commands/command-registry.test.ts\` 的 mockCommandServices 中添加:
   \`\`\`typescript
   getScheduleCooldownStatus: vi.fn(),
   clearScheduleCooldown: vi.fn(),
   \`\`\`

## 触发信息

- CI Monitor 定时任务检测到 main 分支更新
- 新提交: d91a209 (PR #874) 和 dd4c165 (PR #917)
- 测试时间: 2026-03-06

--- a/src/nodes/commands/command-registry.test.ts
+++ b/src/nodes/commands/command-registry.test.ts
@@ -45,6 +45,14 @@ function createMockServices(): CommandServices {
     disableSchedule: () => Promise.resolve(false),
     runSchedule: () => Promise.resolve(false),
     isScheduleRunning: () => false,
+    // Cooldown management (Issue #869)
+    getScheduleCooldownStatus: () => Promise.resolve({
+      isInCooldown: false,
+      lastExecutionTime: null,
+      cooldownEndsAt: null,
+      remainingMs: 0,
+    }),
+    clearScheduleCooldown: () => Promise.resolve(false),
     // Task management methods (Issue #468)
     startTask: () => Promise.resolve({ id: 'task_test', prompt: 'test', status: 'running', progress: 0, chatId: 'oc_test', createdAt: new Date().toISOString(), updatedAt: new Date().toISOString() }),
     getCurrentTask: () => Promise.resolve(null),

--- a/src/schedule/cooldown-manager.test.ts
+++ b/src/schedule/cooldown-manager.test.ts
@@ -4,6 +4,8 @@
  * Issue #869: Cooldown period for scheduled tasks.
  */
 
+/// <reference types="vitest/globals" />
+
 import * as fsPromises from 'fs/promises';
 import * as path from 'path';
 import * as os from 'os';

